src/less/avatar/avatar.less

```diff
@@ -0,0 +1,111 @@
+@import "../variables/variables.less";
+
+.avatar {
+    align-items: center;
+    background-color: var(--color-marketing-teal-background-5);
+    border-radius: 48px;
+    color: var(--color-marketing-teal-foreground-5);
+    display: inline-flex;
+    font-size: @font-size-large-2;
+    font-weight: @font-weight-bold;
+    height: 48px;
+    justify-content: center;
+    overflow: hidden;
+    text-align: center;
```

Comment:
I think `justify-content: center;` is centering things already. I'm not seeing this as necessary. In. fact, excluding it doesn't seem to be making any difference.